servers/mcp-neo4j-cypher/src/mcp_neo4j_cypher/server.py

```diff
@@ -163,27 +79,40 @@ async def write_neo4j_cypher(
     ) -> list[types.TextContent]:
         """Execute a write Cypher query on the neo4j database."""
 
-        if not _is_write_query(query):
+        if not cypher_validation.is_write_query(query):
             raise ValueError("Only write queries are allowed for write-query")
 
-        try:
-            async with neo4j_driver.session(
-                database=os.getenv("NEO4J_DATABASE", "neo4j")
-            ) as session:
-                raw_results = await session.execute_write(_write, query, params)
-                counters_json_str = json.dumps(raw_results._summary.counters.__dict__, default=str)
+        result = await execute_write_query(
+            neo4j_driver, query, params, database, logger, False
```

Comment:
are usually loggers passed to functions in python?
the false should be a named parameter, otherwise the reader doesn't  know what it does